cmd/minishift/cmd/ssh.go

```diff
@@ -34,9 +38,13 @@ var sshCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
+
 		err := cluster.CreateSSHShell(api, args)
 		if err != nil {
-			atexit.ExitWithMessage(1, fmt.Sprintf("Cannot establish SSH connection to the VM: %s", err.Error()))
+			if err.Error() == SshCommunicationError {
```

Comment:
Here we check for the actual outcome to decide the message.
`255` is an ssh error, while anything else is a command error.